pkgs/dart_tooling_mcp_server/lib/src/mixins/analyzer.dart

```diff
@@ -411,27 +441,24 @@ base mixin DartAnalyzerSupport
     description:
         'Get signature help for an API being used at a given cursor '
         'position in a file.',
-    inputSchema: Schema.object(
-      properties: {
-        ParameterNames.uri: Schema.string(
-          description: 'The URI of the file to get signature help for.',
-        ),
-        ParameterNames.line: Schema.int(
-          description: 'The line number of the cursor position.',
-        ),
-        ParameterNames.column: Schema.int(
-          description: 'The column number of the cursor position.',
-        ),
-      },
-      required: [
-        ParameterNames.uri,
-        ParameterNames.line,
-        ParameterNames.column,
-      ],
-    ),
+    inputSchema: _locationSchema,
```

Comment:
does this tool need `readOnly: true` too?

Reply:
It does have it, just a bit hard to tell in the diff